Kudu.Core.Test/Jobs/ScheduleFacts.cs

```diff
@@ -31,12 +31,20 @@ public void ScheduleProvidesNextIntervalCorrectly(string cronExpression, string
 
             var triggeredJobSchedulerLoggerMock = new Mock<TriggeredJobSchedulerLogger>(String.Empty, mockEnvironment.Object, null);
             var schedule = Schedule.BuildSchedule(cronExpression, triggeredJobSchedulerLoggerMock.Object);
-            schedule.SetDateTimeProvider(new TestDateTimeNowProvider(DateTime.Parse("00:00:00 1/1/2015", CultureInfo.InvariantCulture, DateTimeStyles.AssumeLocal)));
+            var now = DateTime.Parse( "00:00:00 1/1/2015" , CultureInfo.InvariantCulture , DateTimeStyles.AssumeLocal );
+            schedule.SetDateTimeProvider( new TestDateTimeNowProvider( now ) );
 
             DateTime lastSchedule = DateTime.Parse(lastScheduleStr, CultureInfo.InvariantCulture, DateTimeStyles.AssumeLocal);
             TimeSpan actualNextSchedule = schedule.GetNextInterval(lastSchedule, ignoreMissed);
 
             TimeSpan expectedNextInterval = TimeSpan.Parse(expectedNextIntervalStr);
+
+            var nowTimeOffset = new DateTimeOffset( now ).Offset;
+            var expectedTimeOffset = new DateTimeOffset( now.Add( TimeSpan.Parse( expectedNextIntervalStr ) ) ).Offset;
+
+            // Adjust the expected value for DST transition on March 11th
```

Comment:
I'd leave this comment above where it was on the test data, since it applies to that test input